dev-requirements.txt

```diff
@@ -21,8 +21,7 @@ lm_eval
 diskcache
 pycocotools
 tqdm
-
-# Custom CUDA Extensions
+git+https://github.com/NVlabs/COAT.git#subdirectory=coat/optimizer/kernels # Custom CUDA Extensions
```

Comment:
Don't add this. CPU runner will fail to build CUDA extension. We will just test this locally.